[2024-05-04T11:32:50.592+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-04T11:32:50.751+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl.gcs_to_bq_customers scheduled__2024-05-03T00:00:00+00:00 [queued]>
[2024-05-04T11:32:50.954+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl.gcs_to_bq_customers scheduled__2024-05-03T00:00:00+00:00 [queued]>
[2024-05-04T11:32:50.974+0000] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2024-05-04T11:32:51.163+0000] {taskinstance.py:2327} INFO - Executing <Task(GCSToBigQueryOperator): gcs_to_bq_customers> on 2024-05-03 00:00:00+00:00
[2024-05-04T11:32:51.359+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=7996) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-04T11:32:51.360+0000] {standard_task_runner.py:63} INFO - Started process 8079 to run task
[2024-05-04T11:32:51.361+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl', 'gcs_to_bq_customers', 'scheduled__2024-05-03T00:00:00+00:00', '--job-id', '156', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpdebes3_i']
[2024-05-04T11:32:51.371+0000] {standard_task_runner.py:91} INFO - Job 156: Subtask gcs_to_bq_customers
[2024-05-04T11:32:51.771+0000] {task_command.py:426} INFO - Running <TaskInstance: etl.gcs_to_bq_customers scheduled__2024-05-03T00:00:00+00:00 [running]> on host d4c1d9b975b8
[2024-05-04T11:32:54.791+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl' AIRFLOW_CTX_TASK_ID='gcs_to_bq_customers' AIRFLOW_CTX_EXECUTION_DATE='2024-05-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-03T00:00:00+00:00'
[2024-05-04T11:32:54.791+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-04T11:32:54.831+0000] {connection.py:274} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-05-04T11:32:54.899+0000] {base.py:84} INFO - Using connection ID 'my_gcp_conn' for task execution.
[2024-05-04T11:32:54.904+0000] {gcs_to_bigquery.py:379} INFO - Using existing BigQuery table for storing data...
[2024-05-04T11:32:55.556+0000] {bigquery.py:2328} INFO - Project is not included in destination_project_dataset_table: order.olist_customers_dataset; using project "project-capstone-422211"
[2024-05-04T11:32:55.651+0000] {gcs_to_bigquery.py:383} INFO - Executing: {'load': {'autodetect': True, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'project-capstone-422211', 'datasetId': 'order', 'tableId': 'olist_customers_dataset'}, 'sourceFormat': 'CSV', 'sourceUris': ['gs://storage-capstone/olist_customers_dataset.csv'], 'writeDisposition': 'WRITE_TRUNCATE', 'ignoreUnknownValues': False, 'skipLeadingRows': None, 'fieldDelimiter': ',', 'quote': None, 'allowQuotedNewlines': False, 'encoding': 'UTF-8'}}
[2024-05-04T11:32:55.656+0000] {bigquery.py:1613} INFO - Inserting job ***_etl_gcs_to_bq_customers_2024_05_03T00_00_00_00_00_13c6c6ee0ceea62c5f8c8f4f015f94a3
[2024-05-04T11:33:35.675+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-04T11:33:35.683+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=etl, task_id=gcs_to_bq_customers, execution_date=20240503T000000, start_date=20240504T113250, end_date=20240504T113335
[2024-05-04T11:33:35.746+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-04T11:33:35.767+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-04T11:33:35.774+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
